packages/ui/src/theme/tokens/components/heading.ts

```diff
@@ -1,4 +1,14 @@
-export const heading = {
+export interface HeadingTokens {
+  color: any;
+  lineHeight: any;
```

Comment:
Should `lineHeight` be a property of the number/"size" keys? Does it need to scale with `fontSize` increases?

Reply:
lineHeights in the theme are unit-less values so they scale with fontSize by default